---
layout: page
title: Software 
subtitle: Well-implemented awesomeness 
---

In addition to papers, we consider high-quality, open-source software to be one of the primary scientific outputs of our work.  Here you will find links to our lab's software and a brief description of each tool.

#### Salmon

Salmon is an accurate, fast and *model-aware* tool for transcript abundance estimation using RNA-seq data.  It achieves its combination of speed and accuracy through a novel approach to read mapping (quasi-mapping) and a dual-phase inference algorithm.

[Source](https://github.com/COMBINE-lab/salmon)

#### RapMap

Rapid sensitive and accurate transcriptome analysis via quasi-mapping.

[Source](https://github.com/COMBINE-lab/RapMap)

#### RapClust

Accurate, lightweight clustering of de novo transcriptomes using fragment equivalence classes.

[Source](https://github.com/COMBINE-lab/RapClust)

#### Matryoshka

Automated discovery of hierarchically-structured chromatin domains.

[Source](https://github.com/COMBINE-lab/Matryoshka)

#### lcmlKin 

Maximum likelihood relatedness estimation from low-coverage sequencing data.

[Source](https://github.com/COMBINE-lab/maximum-likelihood-relatedness-estimation)

#### Sailfish

Sailfish is a tool for the rapid, alignment-free quantification of transcripts from RNA-seq

[Source](https://github.com/kingsfordgroup/sailfish)

